test.sh

```diff
@@ -16,6 +16,32 @@ function main(){
     else
         run_production_tests;
     fi
+
+    # TODO: currently only work on development settings
+    if [[ $(ls -l docker-compose.yml) == *"development.yml"* ]]; then
+        if [ $RUN_E2E ]; then
+            if [ $RUN_E2E = 'true' ]; then
```

Comment:
Do we have to nest the `if`s or can we use a compound condition with lazy evaluation?